.github/workflows/smoke-test-release-run-ubuntu.yml

```diff
@@ -0,0 +1,101 @@
+name: "Positron: Run smoke tests against latest release"
+
+on:
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  linux:
+    name: Tests on Linux
```

Comment:
I assume this is only ever going to be e2e tests? Maybe label it as such, because I know previously "Tests on Linux" was unit, integration and e2e. I think in the full suite I called them: `e2e-electron-tests` for consistency.

Reply:
ok, yeah, consistency is good.  changed!